Dockerfile

```
@@ -25,11 +25,12 @@ ADD ./cmd/ $APP_ROOT/src/cmd/
 ADD ./pkg/ $APP_ROOT/src/pkg/
 
 ARG SERVER_LDFLAGS
+RUN go mod vendor
 RUN go build -ldflags "${SERVER_LDFLAGS}" ./cmd/timestamp-server
 RUN CGO_ENABLED=0 go build -gcflags "all=-N -l" -ldflags "${SERVER_LDFLAGS}" -o timestamp-server_debug ./cmd/timestamp-server
```

It's probably not a bad idea to optimize this as a single `RUN` statement, since the cachi2 environment gets injected for each call to `RUN`.
```suggestion
RUN go mod vendor \
        go build -ldflags "${SERVER_LDFLAGS}" ./cmd/timestamp-server \
        RUN CGO_ENABLED=0 go build -gcflags "all=-N -l" -ldflags "${SERVER_LDFLAGS}" -o timestamp-server_debug ./cmd/timestamp-server
```